crypto/bn/bn_lib.c

```diff
@@ -132,14 +132,73 @@ int BN_num_bits_word(BN_ULONG l)
     return bits;
 }
 
+/* returns 0xff..ff if i==j , 0x0 otherwise */
+static inline int _ints_are_equal(int i, int j)
```

Comment:
Why not use `constant_time_eq()`?

Reply:
Good catch!!

Thanks @paulidale !

For some reason I don't remember anymore, when I started working on this
`constant_time_eq_int()` was not suitable for my initial needs.

Started working on my slightly different implementation and by the time
the code was refactored several times I did not realize I had removed
the reason that prevented me to use the existing `constant_time_eq_int()`
in the first place!

The fixup commit I pushed should solve also the travis error (used `inline` instead of the `ossl_inline` macro for ansi compliance)